plugins/source/azure/codegen/recipes/web.go

```diff
@@ -20,6 +20,21 @@ func Web() []Resource {
 		mockListResult:           mockDirectResponse,
 	}
 
+	var authSettingsV2Resource = resourceDefinition{
```

Comment:
These changes are manually generated (adding `SiteAuthSettingsV2` as a relation of `app`)